CrustTests/Company.swift

```diff
@@ -14,6 +14,35 @@ class Company {
 
 extension Company: AnyMappable { }
 
+enum CompanyCodingKey: Keypath {
+    case uuid
+    case employees(Set<EmployeeCodingKey>)
```

Comment:
Keep in mind that `enum T: String` is not a `String` it's a `RawRepresentable` with a `RawType: String`